storage/clean/oldDatabaseCleaner.go

```diff
@@ -192,7 +192,9 @@ func (odc *oldDatabaseCleaner) cleanOldEpochs(currentEpoch uint32) error {
 		return nil
 	}
 
+	wasCleaned := false
 	for idx, epoch := range sortedEpochs {
+		wasCleaned = true
```

Comment:
What I mentioned above will not apply if this line, actually should be moved after L207 when we are sure that something has been cleaned and only If err == nil. With current approach this flag will be set always if we have something in sortedEpoch and not only if something was cleaned because if epoch >= epochToDeleteTo then the for will break here but the flag will be set and the L212 will be executed.. Which was the real intention?